coordinape/coordinape
# feat: Fixed Payment Amount

fixes #809 

### Test Plan

1. Enable Fixed Payments in Circle Settings by selecting a token
2. Go to any contributor and set a value for their fixed payment amount
3. Save and make sure the amount is reflected in the main contributors table.

Alternatively can just run cypress test for createUsers/updateUsers
